source/Octopus.Tentacle/Util/PlatformDetection.cs

```diff
@@ -12,9 +13,9 @@ public static class PlatformDetection
         public static class Kubernetes
         {
             /// <summary>
-            /// Indicates if the Tentacle is running inside a Kubernetes cluster.
+            /// Indicates if the Tentacle is running inside a Kubernetes cluster as the Kubernetes Agent. This is done by checking if the namespace environment variable is set
             /// </summary>
-            public static bool IsRunningInKubernetes => bool.TryParse(Environment.GetEnvironmentVariable("OCTOPUS__K8STENTACLE__FORCE"), out var b) && b;
+            public static bool IsRunningAsKubernetesAgent => !string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable(KubernetesConfig.NamespaceVariableName));
```

Comment:
Renamed this for clarity as you can run the normal tentacle in Kubernetes, but we are really using this bool when it's running as the kubernetes agent